server/bundles/io.cloudbeaver.server/src/io/cloudbeaver/server/CBAppConfig.java

```diff
@@ -260,4 +264,34 @@ public boolean isGrantConnectionsAccessToAnonymousTeam() {
         return grantConnectionsAccessToAnonymousTeam;
     }
 
+    // we disable embedded drivers by default and enable it in enabled drivers list
+    // that's why we need so complicated logic for disabling drivers
+    public void updateDisabledDriversConfig(String[] disabledDriversConfig) {
+        Set<String> newDisabledDrivers = Arrays.stream(disabledDriversConfig).collect(Collectors.toSet());
```

Comment:
Weirdly complicated logic. Also changes elements order on every run.

Use Set.of or Collections.addAll over LinkedHashSet instead of streams.

Avoid using local variables with same name as class variables.